crates/solidity/src/tests/cli-tests/tests/common.test.ts

```diff
@@ -142,3 +145,53 @@ describe("Run resolc with source debug information, check LLVM debug-info", () =
         });
     }
 });
+
+
+
+describe("Standard JSON compilation with path options", () => {
+    const contractsDir = path.join(shell.tempdir(), 'contracts-test');
+    const inputFile = path.join(__dirname, '..', 'src/contracts/compiled/1.json');
+
+    beforeAll(() => {
+        shell.mkdir('-p', contractsDir);
+
+        const input = JSON.parse(shell.cat(inputFile).toString());
+
+        Object.entries(input.sources).forEach(([sourcePath, source]: [string, any]) => {
+            const filePath = path.join(contractsDir, sourcePath);
+            shell.mkdir('-p', path.dirname(filePath));
+            shell.ShellString(source.content).to(filePath);
+            // I added this here to ensure that we can check for any files
+            // Not being written to temp dirs BUUUT i can remove it
+            console.log(`Wrote source file: ${sourcePath}`);
```

Comment:
Remove the comment and also the `console.log` statement if it is irrelevant for the test.